src/backend/env/user.rs

```diff
@@ -67,6 +67,21 @@ impl UserFilter {
     }
 }
 
+/// Contains the location of the Avatar image blob.
+#[derive(Clone, Serialize, Deserialize)]
+pub struct Avatar {
+    // The canister id of the bucket canister.
+    pub bucket_id: String,
```

Comment:
Ouch, ouch, ouch 🙈 😂  Do you know that the current buckets implementation a pretty stupid "append only" storage and that if posts are deleted, the space is not freed up? (same would hold for avatars)

Did you think about instead simply storing these blob in the current memory, by creating an ObjectManager for these small blobs? There, both storage and removal is already supported.